app/mailers/employee_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class EmployeeMailer < ApplicationMailer
+  def send_delivery_confirmation_email(order)
```

Comment:
I would personally name it like this:
```suggestion
  def delivery_confirmation_email(order)
```

Reply:
Done! :)